frontend/android/app/build.gradle

```
@@ -70,17 +70,17 @@ android {
 
     flavorDimensions "dummy"
     productFlavors {
-        Bayern { flavor ->
+        bayern { flavor ->
             flavor.getIsDefault().set(true)
             setupApplication(flavor, "bayern")
         }
-        Nuernberg { flavor ->
+        nuernberg { flavor ->
             setupApplication(flavor, "nuernberg")
         }
-        Koblenz { flavor ->
+        koblenz { flavor ->
             setupApplication(flavor, "koblenz")
         }
-        BayernFloss { flavor ->
+        bayernFloss { flavor ->
```

:question: Shouldn't this be 
```suggestion
        bayern-floss { flavor ->
```

---

renamed everything to bayernFloss, so we do not have to struggel with special chars.